scripts/jrelease.java

```diff
@@ -0,0 +1,371 @@
+        executeInteractiveCommand("bash", "-c", "./scripts/changelog.sh extract " + nextReleaseVersion);
+
+        // 16. Tag new release in https://github.com/jkubeio/jkube-website (keep empty
+        // title/description)
+        printStep(16, "Tag new release in https://github.com/jkubeio/jkube-website (keep empty title/description)");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 17. Publish tweet in jkubeio profile
+        printStep(17, "Publish tweet in jkubeio profile");
+        // print the message to tweet:
+        // 🔥 @ECDTools #JKube new release 🚀
+        // 📦 v1.14.0
+        // ✨ Helidon support
+        // ✨ several improvements for helm
+        // ✨ CustomResource fragment improvements
+        // 🐞 Many bug fixes & improvements
+        //
+        // 📢 Please help us spread the word & share your experience @jkubeio
+        //
+        // https://github.com/eclipse/jkube/releases/tag/v1.14.0
+        print("🔥 @ECDTools #JKube new release 🚀");
+        print("📦 v" + nextReleaseVersion);
+        print("✨ new feat");
+        print("✨ another great feature");
+        print("✨ some other improvements");
+        print("🐞 Many bug fixes & improvements");
+        print("");
+        print("📢 Please help us spread the word & share your experience @jkubeio");
+        print("");
+        print("https://github.com/eclipse/jkube/releases/tag/v" + nextReleaseVersion);
+        print("");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 18. Announce the release in Gitter
+        printStep(18, "Announce the release in Gitter");
+        print("🔥 Eclipse JKube new release 🚀");
+        print("📦 v" + nextReleaseVersion);
+        print("✨ new feat");
+        print("✨ another great feature");
+        print("✨ some other improvements");
+        print("🐞 Many bug fixes & improvements");
+        print("");
+        print("📢 Please help us spread the word & share your experience https://twitter.com/jkubeio");
+        print("");
+        print("https://github.com/eclipse/jkube/releases/tag/v" + nextReleaseVersion);
+        print("");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 19. Publish release in https://projects.eclipse.org/projects/ecd.jkube
+        printStep(19, "Publish release in https://projects.eclipse.org/projects/ecd.jkube");
+        print("Press enter to continue");
+        System.in.read();
+
+        // 20. Run the Gradle verification test:
+        // https://github.com/jkubeio/jkube-integration-tests/actions/workflows/smoke-tests.yml
+        printStep(20,
+                "Run the Gradle verification test: https://github.com/jkubeio/jkube-integration-tests/actions/workflows/smoke-tests.yml");
+        print("Press enter to continue");
+        System.in.read();
+
+    }
+
+    private static String retrieveCurrentVersionFromPomFile() {
+        String currentVersion = "1.14-SNAPSHOT";
```

Comment:
Why is it initialized to this value ?

Reply:
you are right, maybe it should just fail if it cannot read